src/task.c

```diff
@@ -68,9 +84,15 @@ init_threadid(task_system *ts)
 
 out:
    debug_assert(tid != INVALID_TID);
+
+   // Sets thread-ID, tracked as thread-local storage.
```

Comment:
This is only true on Linux. It could be implemented differently on another platform.

I would just rephrase this as "Sets thread-ID, for example tracked as thread-local storage."

Reply:
Done.